### PR TITLE
Enable tls handling in installer

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- Add tls cert and key as parameters to the installer

**Related issue(s)**

### CODE REVIEW

File: components/installer/cmd/operator/main.go [hunk 1]
```diff
@@ -43,6 +43,8 @@ func main() {
 	kubeconfig := flag.String("kubeconfig", "", "Path to a kubeconfig file")
 	helmHost := flag.String("helmhost", "tiller-deploy.kube-system.svc.cluster.local:44134", "Helm host")
 	kymaDir := flag.String("kymadir", "/kyma", "Directory where kyma packages will be extracted")
+	tlsKey := flag.String("tlsKey", "/etc/certs/tls.key", "Path to TLS key file")
```
Comment (Tomasz-Smelcerz-SAP) [inline components/installer/cmd/operator/main.go]:
```suggestion
	tlsKey := flag.String("tillerTlsKey", "/etc/certs/tls.key", "Path to TLS key file")
```

File: components/installer/cmd/operator/main.go [hunk 2]
```diff
@@ -43,6 +43,8 @@ func main() {
...
+	tlsCrt := flag.String("tlsCrt", "/etc/certs/tls.crt", "Path to TLS cert file")
```
Comment (Tomasz-Smelcerz-SAP) [inline components/installer/cmd/operator/main.go]:
```suggestion
	tlsCrt := flag.String("tillerTLSCrt", "/etc/certs/tls.crt", "Path to TLS cert file")
```

**Comment by Demonsthere** [comment]
@Tomasz-Smelcerz-SAP Yes, this is a kinda breaking change, and will be merged, when the installation process has been updated. That process is done in https://github.com/kyma-project/kyma/pull/3223

**Pull request merged**